### DISCUSSION TITLE
Should Rumble be working on Gamecube?

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#71] [Q&A] [answered]
Just realised that my controllers have rumble (didn't realise that they did til I just used them on Switch) but that I've never felt a rumble... should rumble work on Gamecube? 

They are 8BitDo so if it should, wonder if it might be somehow related to this issue (https://github.com/darthcloud/BlueRetro/issues/69)?

(nb. there is a small possibility that the games I have played so far just don't use rumble... will try and find a reliable way to test later!)

**Comment by darthcloud** [comment, marked as answer]
If rumble is enable in the web config for that player (It's all disable by default) rumble should work.

Note that rumble is broken in the v1.0-beta, but v0.19.1 is fine.

**Reply by vaderag** [reply]
Ah okay. Perhaps not enabling it is the issue! Will check :) Thanks!

If not, I'll log an issue
